components/language-chooser/react/language-chooser-react-mui/src/LanguageChooser.tsx

```diff
@@ -204,8 +204,6 @@ export const LanguageChooser: React.FunctionComponent<{
                 width: 100%;
                 min-width: 100px;
                 max-width: 436px;
-                // TODO (currently working on it) border interferes with the focus indicator outline
-                // border: 1px solid ${COLORS.greys[3]};
```

Comment:
I put a card in the kanban to ask john about these. His mocks have borders, but they interact weirdly with the outlines showing focus